mulighetsrommet-api/src/main/resources/db/migration/R__tilsagn_admin_view.sql

```diff
@@ -26,12 +20,66 @@ select tilsagn.id,
        arrangor.navn                                                             as arrangor_navn,
        arrangor.slettet_dato is not null                                         as arrangor_slettet,
        gjennomforing.tiltaksnummer                                               as tiltaksnummer,
-       tiltakstype.tiltakskode                                                   as tiltakskode
+       tiltakstype.tiltakskode                                                   as tiltakskode,
+       opprett_to_trinnskontroll_json,
+       annuller_to_trinnskontroll_json
 from tilsagn
          inner join nav_enhet on nav_enhet.enhetsnummer = tilsagn.kostnadssted
          inner join arrangor on arrangor.id = tilsagn.arrangor_id
          inner join gjennomforing on gjennomforing.id = tilsagn.gjennomforing_id
          inner join tiltakstype on tiltakstype.id = gjennomforing.tiltakstype_id
-         left join nav_ansatt nav_ansatt_beslutter on nav_ansatt_beslutter.nav_ident = tilsagn.status_besluttet_av
-         left join nav_ansatt nav_ansatt_endret_av on nav_ansatt_endret_av.nav_ident = tilsagn.status_endret_av;
+         left join lateral (
```

Comment:
Siden "ToTrinnskontroll" er trukket ut i egne queries så hadde det kanskje vært mening å gjøre det samme for data-hentingen også?

Kunne du også tatt en gjennomgang med oss hvordan totrinnskontroll nå er ment å funke? F.eks. om det også skal gjelde for utbetalinger.